### PR TITLE
Added more examples and explanations

### PR DESCRIPTION
**Pull request by JensTrier**
Updated Set-CsTeamsEmergencyCallRoutingPolicy with more examples and add link to Get. Ensured other pages for '-CsTeamsEmergencyCallRoutingPolicy also referenced the Get cmdlet.

Added more explanation to Get-CsOnlineDialOutPolicy and updated syntax

### CODE REVIEW

File: skype/skype-ps/skype/Get-CsOnlineDialOutPolicy.md [hunk 1]
```diff
@@ -19,39 +19,40 @@ Use the `Get-CsOnlineDialOutPolicy` cmdlet to get all the available outbound cal
...
 ### Filter
 ```
-Get-CsOnlineDialOutPolicy [-Filter <String>] [-LocalStore] [<CommonParameters>]
+Get-CsOnlineDialOutPolicy [-Filter <string>] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-In Skype for Business Online, outbound calling restriction policies are used to restrict the type of audio conferencing and end user PSTN calls that can be made by users in your organization. 
-To get all the available policies in your organization run `Get-CSOnlineDialOutPolicy`.
+In Microsoft Teams, outbound calling restriction policies are used to restrict the type of audio conferencing and end user PSTN calls that can be made by users in your organization. The policies applies to all the different PSTN connectivity options for Microsoft Teams - Calling Plan, Direct Routing and Operator Connect.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineDialOutPolicy.md:31]:
```suggestion
In Microsoft Teams, outbound calling restriction policies are used to restrict the type of audio conferencing and end user PSTN calls that can be made by users in your organization. The policies apply to all the different PSTN connectivity options for Microsoft Teams; Calling Plan, Direct Routing, and Operator Connect.
```

File: skype/skype-ps/skype/Grant-CsDialoutPolicy.md [hunk 2]
```diff
@@ -33,7 +33,8 @@ Grant-CsDialoutPolicy [-Group] <string> [[-PolicyName] <string>] [-PassThru] [-R
 ```
 
 ## DESCRIPTION
-In Skype for Business Online and Microsoft Teams, outbound calling restriction policies are used to restrict the type of audio conferencing and end user PSTN calls that can be made by users in your organization.
+In Microsoft Teams, outbound calling restriction policies are used to restrict the type of audio conferencing and end user PSTN calls that can be made by users in your organization. The policies applies to all the different PSTN connectivity options for Microsoft Teams - Calling Plan, Direct Routing and Operator Connect.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Grant-CsDialoutPolicy.md:36]:
```suggestion
In Microsoft Teams, outbound calling restriction policies are used to restrict the type of audio conferencing and end user PSTN calls that can be made by users in your organization. The policies apply to all the different PSTN connectivity options for Microsoft Teams; Calling Plan, Direct Routing, and Operator Connect.
```

File: skype/skype-ps/skype/Grant-CsTeamsEmergencyCallRoutingPolicy.md [hunk 3]
```diff
@@ -43,10 +43,17 @@ This cmdlet assigns a Teams Emergency Call Routing policy to a user, a group of
 
 ### Example 1
 ```powershell
-PS C:> Grant-CsTeamsEmergencyCallRoutingPolicy -Identity user1 -PolicyName TestECRP
+Grant-CsTeamsEmergencyCallRoutingPolicy -Identity user1 -PolicyName Test
 ```
 
-This example assigns a Teams Emergency Call Routing policy(TestECRP) to a user(user1)
+This example assigns a Teams Emergency Call Routing policy(Test) to a user(user1).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Grant-CsTeamsEmergencyCallRoutingPolicy.md:49]:
```suggestion
This example assigns a Teams Emergency Call Routing policy (Test) to a user (user1).

```

File: skype/skype-ps/skype/Grant-CsTeamsEmergencyCallRoutingPolicy.md [hunk 4]
```diff
@@ -43,10 +43,17 @@ This cmdlet assigns a Teams Emergency Call Routing policy to a user, a group of
...
 
-This example assigns a Teams Emergency Call Routing policy(TestECRP) to a user(user1)
+This example assigns a Teams Emergency Call Routing policy(Test) to a user(user1).
+
+### Example 2
+```powershell
+Grant-CsTeamsEmergencyCallRoutingPolicy -Group sales@contoso.com -Rank 10 -PolicyName Test
+```
+
+This example assigns the Teams Emergency Call Routing policy(Test) to the members of the group sales@contoso.com.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Grant-CsTeamsEmergencyCallRoutingPolicy.md:56]:
```suggestion
This example assigns the Teams Emergency Call Routing policy (Test) to the members of the group sales@contoso.com.

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @JensTrier, kindly commit the suggested changes. Many thanks.

**Comment by JensTrier** [comment]
> Copy review - @JensTrier, kindly commit the suggested changes. Many thanks.

Thanks @JohanFreelancer9 & done! Happy holidays and thanks for your awesome help during 2022.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by get-itips — APPROVED** [review]

**Comment by JohanFreelancer9** [comment]
> > Copy review - @JensTrier, kindly commit the suggested changes. Many thanks.
> 
> Thanks @JohanFreelancer9 & done! Happy holidays and thanks for your awesome help during 2022.

Thank you @JensTrier. Always a pleasure. Happy holidays to you too!

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**